File: src/main/java/com/ggardet/SpringModulithApplication.java
package com.ggardet;

import org.springframework.boot.SpringApplication;
import org.springframework.modulith.Modulith;
import org.springframework.scheduling.annotation.EnableAsync;
import org.springframework.scheduling.annotation.EnableScheduling;

@Modulith
@EnableAsync
@EnableScheduling
public class SpringModulithApplication {
    public static void main(String[] args) {
        SpringApplication.run(SpringModulithApplication.class, args);
    }
}

File: src/main/kotlin/com/ggardet/SpringModulithApplication.kt
package com.ggardet

import org.springframework.boot.runApplication
import org.springframework.modulith.Modulith
import org.springframework.scheduling.annotation.EnableAsync
import org.springframework.scheduling.annotation.EnableScheduling

@Modulith
@EnableAsync
@EnableScheduling
class SpringModulithApplication

fun main(args: Array<String>) {
    runApplication<SpringModulithApplication>(*args)
}
